packages/components/src/Chip/Chip.css

```diff
@@ -129,31 +128,32 @@
 */
 
 .subtle:focus-visible {
-  --chip--gradient-color-variation--hover: var(--color-surface);
+  /* this one looks suspicious */
+  /* why does this not match the hover style? */
+  --chip-base-hover-bg-color: var(--color-surface);
 }
 
 .subtle:hover {
-  border: var(--border-base) solid var(--chip-base-bg);
-  background-color: var(--chip-base-bg);
-  --chip--gradient-color-variation--hover: var(--chip-base-bg);
+  --chip-border-color: var(--chip-base-bg-color);
+  --chip-base-hover-bg-color: var(--color-surface--background);
 }
 
 .invalid.disabled,
 .invalid.disabled:hover,
 .invalid.disabled:focus-visible {
-  background-color: var(--color-disabled--secondary);
-  cursor: auto;
+  --chip-base-bg-color: var(--color-disabled--secondary);
+  --chip-base-hover-bg-color: var(--color-disabled--secondary);
+  cursor: default;
 }
 
 .disabled,
 .disabled:hover,
 .disabled:focus-visible {
-  border: var(--border-base) solid var(--color-disabled--secondary);
+  --chip-border-color: var(--color-disabled--secondary);
   color: var(--color-disabled);
-  background-color: var(--color-disabled--secondary);
-  cursor: auto;
-  --chip--gradient-color-variation: var(--color-disabled--secondary);
-  --chip--gradient-color-variation--hover: var(--color-disabled--secondary);
+  --chip-base-bg-color: var(--color-disabled--secondary);
+  --chip-base-hover-bg-color: var(--color-disabled--secondary);
+  cursor: default;
```

Comment:
This is more of a design question, but should this, perhaps, be `cursor: not-allowed`? We are using this typo of cursor for disabled buttons, checkboxes and radio and switches.

Reply:
good question

right now on master with the old `Chips` setup, if you disable a chip you still get a `cursor: pointer` which is probably not the right thing to do

![image](https://github.com/user-attachments/assets/d55e09db-98c3-4a49-910d-ce06a38a2f9f)

Reply:
@chris-at-jobber any preference for the cursor style on disabled `Chip`s?

Reply:
`not-allowed` would be preferred!